google/cloud/spanner_v1/pool.py

```diff
@@ -162,7 +162,7 @@ def bind(self, database):
         """Associate the pool with a database.
 
         :type database: :class:`~google.cloud.spanner_v1.database.Database`
-        :param database: database used by the pool:  used to create sessions
+        :param database: database used by the pool to used to create sessions
```

Comment:
nit: database used by the pool to create sessions ?

Reply:
Great spotting! Fixed :)